tests/online_migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -31,7 +31,51 @@ sub run {
         # set blank screen to be never for current session
         script_run("gsettings set org.gnome.desktop.session idle-delay 0");
         become_root;
+        if (check_var('HDDVERSION', '12')) {
```

Comment:
does this specifically only apply for SLE 12 GA or for <= SLE 12 GA?

`if (!sle_version_at_least('12', version_variable => 'HDDVERSION')` should work in the latter case